tests/test_bayes_factor.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import arviz as az
```

Comment:
Don't use arviz! you need to  arviz-base, stats or plots. 
```suggestion
from arviz_base import from_dict
```